traitcuration/traits/models.py

```diff
@@ -0,0 +1,91 @@
+from django.db import models
+
+
+class Trait(models.Model):
+    name = models.CharField(max_length=200)
+    current_mapping = models.ForeignKey(
+        'Mapping', on_delete=models.SET_NULL, null=True, blank=True)
+    status = models.CharField(max_length=50, choices=[
+        ('current', 'Current'),
+        ('unmapped', 'Unmapped'),
+        ('obsolete', 'Obsolete'),
+        ('deleted', 'Deleted'),
+        ('awaiting_review', 'Awaiting Review'),
+        ('needs_import', 'Needs Import'),
+        ('awaiting_import', 'Awaiting Import'),
+        ('needs_creation', 'Needs Creation'),
+        ('awaiting_creation', 'Awaiting Creation')])
+    number_of_source_records = models.IntegerField(blank=True, null=True)
+    timestamp_added = models.DateTimeField(auto_now=True)
+    timestamp_updated = models.DateTimeField(auto_now_add=True)
+
+    def __str__(self):
+        return self.name
+
+
+class User(models.Model):
+    username = models.CharField(max_length=100, unique=True)
+    email = models.EmailField(unique=True)
+    role = models.CharField(max_length=20, blank=True, null=True)
+
+    def __str__(self):
+        return self.username
+
+
+class OntologyTerm(models.Model):
+    curie = models.CharField(max_length=50, blank=True, null=True, unique=True)
+    uri = models.URLField(null=True, blank=True, unique=True)
+    label = models.CharField(max_length=200)
+    status = models.CharField(max_length=50, choices=[
+        ('current', 'Current'),
+        ('obsolete', 'Obsolete'),
+        ('deleted', 'Deleted'),
+        ('unmapped', 'Unmapped'),
+        ('needs_import', 'Needs Import'),
+        ('awaiting_import', 'Awaiting Import'),
+        ('needs_creation', 'Needs Creation'),
+        ('awaiting_creation', 'Awaiting Creation')])
```

Comment:
To avoid code duplication, I think this list should be moved to a variable, and then the list of statuses for the Trait should be this variable _plus_ "awaiting review".